.github/pull_request_template.md

```diff
@@ -1,10 +1,38 @@
-This PR implements the following **changes:**
+## summary
 
-* insert your summary of your request here, add more bullets as needed
+Basic description of work done.
 
+[Link to Preview]()
 
-**URL / Link to page**
+## steps to recreate
```

Comment:
We can improve clarity by using `How to test` instead because this is a solution. I would also place this after the solution.